packages/unmock-node/src/__tests__/backend/index.test.ts

```diff
@@ -1,8 +1,8 @@
 import axios from "axios";
 import { removeCodes } from "openapi-refinements";
 import * as path from "path";
-import { Service, sinon, transform, UnmockPackage } from "../../";
-import NodeBackend from "../../backend/node";
+import { Service, sinon, transform, UnmockPackage } from "unmock-core";
```

Comment:
Obviously not for this PR, but maybe we can retire `UnmockPackage`? :thinking:

Reply:
Or well, more like rename it? Or... yeah not sure, actually maybe since we're planning for a browser version and such, it might finally earn its place..?

Reply:
Yep I think it's fine for it to exist as the top-level API, ensuring all packages export an instance of the same class.